codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -161,31 +160,10 @@ public B gracefulShutdownTimeoutMillis(long gracefulShutdownTimeoutMillis) {
         }
 
         /**
-         * Determine if the encoder should queue frames to honor the value set by
-         * {@link #encoderMaxConcurrentStreams(int)}.
+         * Determine if the encoder should queue frames
```

Comment:
Determine if the encoder should queue frames if the maximum number of concurrent streams would otherwise be exceeded.